migrations/V148__System_Intake_Edits_Requested_Add_Step_To_Action.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE actions ADD COLUMN step system_intake_step;
```

Comment:
@mynar7 , just to confirm, is the expected behavior that this field should be updated with whatever the step was when an action was taken? When I request edits locally, the `step` column is null in the action table, instead of saying the step that the request was on.

Reply:
I believe it should be the target step, but yes, it looks like I forgot to update it.

Addressed in [4aa9736](https://github.com/CMSgov/easi-app/pull/2129/commits/4aa97368d9b0789d99741ee6968dadc88b103064)